new-frontiers-14/frontier-station-14
# Faction Revisions

## About the PR



A bit of faction revision.

1. Clarpy remains a pirate, but now the pirates are pirates.
2. Pirates are now _only_ members of PiratesNF, not their own species.
3. PetsNT (e.g. emotional support pets, Clippy) isn't inherently hostile to anyone save mice.  They might retaliate, but Clippy won't go for the jugular off the bat, ever.  Keep in mind, Clarpy _will_.
4. Cappy is now a member of NanoTrasen (congratulations).
5. **CDETs only target contraband, they won't stunlock a pirate without reason (note: faction attribution is still busted, that's to be reviewed separately).**
6. Baby dragons aren't dragons, they can be brought onto Frontier Outpost, or ghost without attacking anybody.

Edit: also gave the cats accents, because why not, and fixed a missing localization string for hyperzine.

## Why / Balance


https://discord.com/channels/1123826877245694004/1315125515266555954

There's another issue I can't find that complains about pirates being targeted without items - mentioning this because (with an NPCFactionMember fix) this shouldn't be an issue, even now - the CDET will not target pirates without cause.

## How to test


1. Spawn a baby dragon on Frontier, it shouldn't turn to ash.
2. Spawn Clugg on Frontier, it shouldn't turn to ash.
3. Spawn Clugg and a dino in a room together, they should be friendly enough.
4. Spawn in as a pirate, first mate and captain, check your NPCFactionComponent, you should be part of the PiratesNF faction.
5. Give Clarpy and Clippy a hug, they shouldn't maul you.
6. Spawn Cappy - he _should_ maul you.

## Media


## Requirements



## Breaking changes


**Changelog**


:cl:
- fix: Clarpy no longer attacks pirates, Clippy no longer attacks pirates.
- tweak: The CDET now only targets entities wearing contraband clothing.
- tweak: Clugg now has a caveman accent; Clarpy, a pirate accent; Cappy, a southern accent if cognizined.
